api/node.go

```diff
@@ -675,3 +675,68 @@ func rebalanceNodesHandler(w http.ResponseWriter, r *http.Request, t auth.Token)
 	fmt.Fprintf(writer, "Units successfully rebalanced!\n")
 	return nil
 }
+
+type infoNodeResponse struct {
+	Node   provision.NodeSpec    `json:"node"`
+	Status healer.NodeStatusData `json:"status"`
+	Units  []provision.Unit      `json:"units"`
+}
+
+// title: node info
+// path: /node/{address}
+// method: GET
+// produce: application/json
+// responses:
+//   200: Ok
+//   400: Invalid data
+//   404: Not found
+func infoNodeHandler(w http.ResponseWriter, r *http.Request, t auth.Token) error {
+	address := r.URL.Query().Get(":address")
+	if address == "" {
+		return errors.Errorf("Node address is required.")
+	}
+	if err := validateNodeAddress(address); err != nil {
```

Comment:
I don't think this validate call is necessary. Actually it can cause errors if the address is not a URL as is the case for kubernetes provisioner nodes.